config/routes.rb

```diff
@@ -1,3 +1,3 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  resources :restaurants
```

Comment:
Remember that you can (and should) scope the routes you create using the 'only:' keyword